### ISSUE TITLE
null values for numbers in package.json cause unhandled error

### ISSUE THREAD

**Issue by morille** [jyapayne/Web2Executable#195]
First, many thanks for this application.
When I open a project, the "readonly" mode is on by default, then when I toggle it it just enable the 2 first tabs (App settings / Windows settings), not the other one nor the export toolbar at the bottom.
The last version I can use is 0.5.3b.
Have a nice day.

**Comment by jyapayne** [comment]
Hi morille, can you give me a link to the project that exhibits this behavior? The projects that I have tested on (including an empty folder) do not behave like this.

**Comment by morille** [comment]
Hi jyapayne, I tried to reduce my package.json until the application behaves correctly and I found what caused this error: The entries "window.max_width" and "window.max_height" in my package.json were both set to null to allow my app to have any size.
When I replace null by 0, everything works fine both in Web2Executable and NWJS.
Take any project and set these values to null and Web2Executable will hang during project opening and refuse to unlock the interface.
